Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -424,6 +424,12 @@ extension ChartDataSet: RandomAccessCollection {
 
 // MARK: RangeReplaceableCollection
 extension ChartDataSet: RangeReplaceableCollection {
+    public func replaceSubrange<C>(_ subrange: Swift.Range<Index>, with newElements: C) where C : Collection, Element == C.Element
+    {
+        entries.replaceSubrange(subrange, with: newElements)
+        notifyDataSetChanged()
+    }
+    
```

Comment:
This was the fix.

Reply:
what's the original issue here? It has been too long; sorry

I would like to get this merged asap, so bear with me my direct asking

Reply:
also - if we are compiling with old Swift, like 5.0 or 5.4, will this cause a regression? Not fulling understanding the issue

Reply:
basically we usually just support latest one; but given this issue is totally different 5.6/5.7, I'm not wure what would happen in the field. 

Would @avaialble(iOS 16) , (macOS, 13) or something similar solve it?

Reply:
I was trying to compile on Xcode beta3, howerver, it still throws 
```
Type 'ChartDataSet' does not conform to protocol 'RangeReplaceableCollection'
Unavailable instance method 'replaceSubrange(_:with:)' was used to satisfy a requirement of protocol 'RangeReplaceableCollection'
```
which Xcode beta you are building?